src/Servers/IIS/IntegrationTesting.IIS/src/IISDeployer.cs

```diff
@@ -391,11 +391,8 @@ private void Stop()
                 // Stop all app pools
                 foreach (var appPool in serverManager.ApplicationPools)
                 {
-                    if (appPool.State != ObjectState.Stopped && appPool.State != ObjectState.Stopping)
-                    {
-                        var state = appPool.Stop();
-                        Logger.LogInformation($"Stopping pool, state: {state}");
-                    }
+                    var state = site.Stop();
```

Comment:
Something wrong w/ conflict resolution here I assume. Are parts of b2830adc665b and 9d121881aaf4 causing problems❔